src/commonMain/kotlin/baaahs/Pinky.kt

```diff
@@ -82,6 +82,11 @@ class Pinky(
         }
 
         GlslBase.plugins.add(SoundAnalysisPlugin(soundAnalyzer))
+
+        // save these if we want to explicitly unregister them or update their TXT records later
+        mdns.register(link.myHostname, "_sparklemotion-pinky", "_udp", Ports.PINKY, "local.", mutableMapOf(Pair("MAX_UDP_SIZE", "1450")))
+        mdns.register(link.myHostname, "_sparklemotion-pinky", "_tcp", Ports.PINKY_UI_TCP, "local.")
+        mdns.listen("_sparklemotion-brain", "_udp", "local.", MdnsBrainListenHandler())
```

Comment:
Would be nice to make this strings constants for maintainability and avoiding of fat fingering these in the future